index.html

```diff
@@ -1,13 +1,97 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
-
-    <link href="styles.css" rel="stylesheet" />
-
-    <title>Rocket Academy</title>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>YY's Portfolio</title>
+    <link rel="icon" href="images/panda favicon.png" />
+    <link rel="stylesheet" href="nav-bar.css" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="about-yy.css" />
+    <link rel="stylesheet" href="projects.css" />
+    <link rel="stylesheet" href="contact-me.css" />
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Arima+Madurai:wght@100;200;300;400;500;700;800;900&family=Roboto:wght@400;500;700&display=swap');
```

Comment:
personally i prefer to have my css downloaded via a `<link>`, but this is fine too. may be a bit harder to track if your code base grows and you end up creating a lot of styles here in `index.html`